.editorconfig

```diff
@@ -34,3 +34,11 @@ indent_size = 4
 
 csharp_prefer_simple_using_statement = true:warning
 csharp_style_prefer_switch_expression = true:warning
+
+dotnet_style_qualification_for_field = true:warning
+dotnet_style_qualification_for_property = false:warning
+dotnet_style_qualification_for_method = false:warning
+dotnet_style_qualification_for_event = false:warning
```

Comment:
Sorry if I wasn't clear earlier. These should all be true.

Reply:
done